### PR TITLE
Update downloader.sh

### LINKED ISSUE #29: -fo tag to open folder of download rather than the pdf
Should read input tag and make changes to downloader.sh

### PR DESCRIPTION
**Pull request by lokeshkvn**
Adds open folder by -fo tag

fixes:#29

### CODE REVIEW

File: BASH/downloader.sh [hunk 1]
```diff
@@ -96,6 +100,8 @@ do
 	path=$(pwd)
 	echo -e "Your downloaded file is in this path:\n" $path
 	cd ..
-	gnome-open chap$chapno.pdf
-	rm -rf $chap
```
Comment (pandusonu2) [inline BASH/downloader.sh:100]: Why did you delete these lines?

File: BASH/downloader.sh [hunk 2]
```diff
@@ -96,6 +100,8 @@ do
...
+        if [ "$x" != "" ]; then
+        nautilus $path
+        break
```
Comment (pandusonu2) [inline BASH/downloader.sh:105]: It shouldn't break here.

**Review by pandusonu2 — CHANGES_REQUESTED** [review]

File: BASH/downloader.sh (hunk 1 above)
Comment (lokeshkvn) [reply BASH/downloader.sh:100]: Otherwise it will open pdf,  so i deleted these lines

File: BASH/downloader.sh (hunk 2 above)
Comment (lokeshkvn) [reply BASH/downloader.sh:105]: ok
